--- pyproject.toml.old	2026-09-20 18:22:22.611868823 -0500
+++ pyproject.toml	2026-09-20 18:22:22.629774542 -0500
@@ -7,8 +7,7 @@
 version = "3.20.0"
 description = "Backup and Recovery Manager for PostgreSQL"
 readme = "README.rst"
-license = "GPL-3.0-only"
-license-files = ["LICENSE"]
+license = {text = "GPL-3.0-only"}
 requires-python = ">=3.12"
 authors = [
   {name = "EnterpriseDB", email = "barman@enterprisedb.com"},
@@ -34,6 +33,9 @@
   "python-dateutil",
 ]
 
+[tool.setuptools]
+license-files = ["LICENSE"]
+
 [project.urls]
 Homepage = "https://www.pgbarman.org/"
 Documentation = "https://docs.pgbarman.org/"
