--- Makefile.old	2026-06-29 11:39:33.565191160 +0300
+++ Makefile	2026-06-29 11:39:38.770750031 +0300
@@ -21,20 +21,6 @@
 PG_CONFIG ?= pg_config
 PGXS := $(shell $(PG_CONFIG) --pgxs)
 
-install16:
-	$(MAKE) clean && $(MAKE) all PG_CONFIG=/usr/lib/postgresql/16/bin/pg_config && $(MAKE) install PG_CONFIG=/usr/lib/postgresql/16/bin/pg_config
-
-install17:
-	$(MAKE) clean && $(MAKE) all PG_CONFIG=/usr/lib/postgresql/17/bin/pg_config && $(MAKE) install PG_CONFIG=/usr/lib/postgresql/17/bin/pg_config
-
-install18:
-	$(MAKE) clean && $(MAKE) all PG_CONFIG=/usr/lib/postgresql/18/bin/pg_config && $(MAKE) install PG_CONFIG=/usr/lib/postgresql/18/bin/pg_config
-
-installall:
-	$(MAKE) install16
-	$(MAKE) install17
-	$(MAKE) install18
-
 # Detect OS
 UNAME_S := $(shell uname -s)
 UNAME_M := $(shell uname -m)
