--- pyproject.toml.old	2026-09-20 20:20:12.912660990 -0500
+++ pyproject.toml	2026-09-20 20:20:12.933597644 -0500
@@ -6,7 +6,7 @@
 authors = [
     { name = "Gregory Szorc", email = "gregory.szorc@gmail.com" },
 ]
-license = "BSD-3-Clause"
+license = {text = "BSD-3-Clause"}
 requires-python = ">=3.9"
 classifiers = [
     "Development Status :: 5 - Production/Stable",
@@ -18,7 +18,6 @@
     "Programming Language :: Python :: 3.12",
     "Programming Language :: Python :: 3.13",
     "Programming Language :: Python :: 3.14",
-    "Programming Language :: Python :: Free Threading :: 1 - Unstable",
 ]
 keywords = ["zstandard", "zstd", "compression"]
 dependencies = []
