From 296452e64b9ba0bedad8a9352ea299e0c902d8d7 Mon Sep 17 00:00:00 2001
From: jesperpedersen <jesper.pedersen@comcast.net>
Date: Sun, 5 Jan 2025 17:21:25 -0500
Subject: [PATCH] Check readlink()

---
 src/libpgmoneta/utils.c | 17 ++++++++++++++++-
 1 file changed, 16 insertions(+), 1 deletion(-)

diff --git a/src/libpgmoneta/utils.c b/src/libpgmoneta/utils.c
index 8032d0b4..0fc61306 100644
--- a/src/libpgmoneta/utils.c
+++ b/src/libpgmoneta/utils.c
@@ -2783,6 +2783,10 @@ pgmoneta_get_symlink(char* symlink)
 
    memset(&link[0], 0, sizeof(link));
    size = readlink(symlink, &link[0], sizeof(link));
+   if (size == -1)
+   {
+      goto error;
+   }
    link[size + 1] = '\0';
 
    if (strlen(&link[0]) == 0)
@@ -2812,6 +2816,7 @@ pgmoneta_is_symlink_valid(char* path)
 {
    char* link = NULL;
    struct stat buf;
+   ssize_t size;
    bool ret = false;
 
    if (lstat(path, &buf) == 0)
@@ -2819,7 +2824,11 @@ pgmoneta_is_symlink_valid(char* path)
       link = malloc(buf.st_size + 1);
       memset(link, 0, buf.st_size + 1);
 
-      readlink(path, link, buf.st_size + 1);
+      size = readlink(path, link, buf.st_size + 1);
+      if (size == -1)
+      {
+         goto error;
+      }
       link[buf.st_size] = '\0';
 
       if (stat(link, &buf) == 0)
@@ -2831,6 +2840,12 @@ pgmoneta_is_symlink_valid(char* path)
    free(link);
 
    return ret;
+
+error:
+
+   free(link);
+
+   return false;
 }
 
 int
