From fdfeab792e8a1d53993242063131dcd0b6ea4f80 Mon Sep 17 00:00:00 2001
From: jesperpedersen <jesper.pedersen@comcast.net>
Date: Wed, 18 Dec 2024 04:03:53 -0500
Subject: [PATCH] Check for NULL

---
 src/libpgmoneta/walfile.c | 14 ++++++++++----
 1 file changed, 10 insertions(+), 4 deletions(-)

diff --git a/src/libpgmoneta/walfile.c b/src/libpgmoneta/walfile.c
index 2a97dd98..7e935e3d 100644
--- a/src/libpgmoneta/walfile.c
+++ b/src/libpgmoneta/walfile.c
@@ -360,8 +360,11 @@ pgmoneta_describe_walfile(char* path, enum value_type type, char* output, bool q
 
    if (output != NULL)
    {
-      fflush(out);
-      fclose(out);
+      if (out != NULL)
+      {
+         fflush(out);
+         fclose(out);
+      }
    }
 
    free(tmp_wal);
@@ -374,8 +377,11 @@ pgmoneta_describe_walfile(char* path, enum value_type type, char* output, bool q
 
    if (output != NULL)
    {
-      fflush(out);
-      fclose(out);
+      if (out != NULL)
+      {
+         fflush(out);
+         fclose(out);
+      }
    }
 
    free(tmp_wal);
